src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,11 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
```

Comment:
Consider adding an `aria-label` or `title` attribute to the anchor tag to improve accessibility. 🎉